src/apipkg/__init__.py

```diff
@@ -83,6 +91,22 @@ def importobj(modpath, attrname):
     return retval
 
 
+def synchronized(wrapped_function):
```

Comment:
lets make this private, i might want to inline it at a later point when spelling the global lock different

Reply:
Ok, decorator renamed to indicate that it's a private implementation detail. Anything else?